_data/simple-icons.json

```
@@ -4396,7 +4396,12 @@
         {
             "title": "dm",
             "hex": "002878",
-            "source": "https://www.dm.de"
+            "source": "https://www.dmtech.tech",
```

The main website still applies - dmtech is just the technology branch of the business.

```suggestion
            "source": "https://www.dm.de",
```